californium-proxy/pom.xml

```diff
@@ -17,9 +17,7 @@
 	<description>Cross-proxy module</description>
 
 	<properties>
-		<httpcomponents.version>4.2.5</httpcomponents.version>
-		<httpcomponents.version.lowerbound>4.2</httpcomponents.version.lowerbound>
-		<httpcomponents.version.upperbound>5</httpcomponents.version.upperbound>
+		<httpasyncclient.version>4.1.3</httpasyncclient.version>
```

Comment:
ok, I have done some research regarding already approved CQs for HttpComponents artifacts at Eclipse. It seems that using version 4.1.2 of the async client will induce the least effort regarding dependencies while still being sufficiently up to date. So please change the version to 4.1.2.

Reply:
we will then need to make a lot of additions to the maven bundle plugin configuration in order to specify the required versions of packages to import. Sadly but truly it seems that the arbitrary HttpComponents artifacts do not contain separate *sub-trees* of packages which would allow us to use some reasonable wild cards but instead contain a wild mix of complementary packages at all levels of the package hierarchy (sigh). I will take take of that once this PR is integrated, though, so you do not need to bother ...

Reply:
Done.